packages/core/src/domain/patient.ts

```diff
@@ -40,7 +40,7 @@ export type DriversLicense = {
   state: USState;
 };
 
-export const genderAtBirthTypes = ["F", "M"] as const;
+export const genderAtBirthTypes = ["F", "M", "UN"] as const;
```

Comment:
UK is one of the three allowed values for gender in IHE. If an endpoint says a gender is UK, thats very different than it being undefined IMO. 

Answering this I realized that gender should have been required anyway. I updated things to reflect that and that simplifies things a lot

Reply:
I think we need to address this holistically. I put this [this Notion page](https://www.notion.so/metriport/Gender-Mapping-41bd8ea7605f486dbe63fce826b1a8fe?pvs=4) together and will bring this to #engineering.

For this PR, in specific:
- Assuming we're meaning `UN` is "Undifferentiated", right?
- On [this line](https://github.com/metriport/metriport/pull/2032/files#diff-6db2ca6d1bb2d596870b18ee9e3671a95a06b53703a41955d6b31abe44509498R19), we're mapping `UN` to "unknown", but according to the [CDA<>FHIR spec](https://build.fhir.org/ig/HL7/ccda-on-fhir/branches/master/ConceptMap-CF-AdministrativeGender.html), `UN` maps to "other"
- it would be good to update the types to specify the word we mean by each acronym, e.g.:
   - `GenderDTO`'s `U`, says its "unspecified" but according to [this mapping](https://github.com/metriport/metriport/pull/2032/files#diff-8ff2297583dba4f8bf6849bc82c683b308de84bb36227b9b0499cd84e6aa8ba7R21) it should mean "Undifferentiated", right?
   - `genderAtBirthTypes`